AdyenCard/Components/Card/CardComponentConfiguration.swift

```diff
@@ -87,29 +87,29 @@ public protocol AnyCardComponentConfiguration {
 extension CardComponent {
 
     /// The mode of address form of card component
-    public enum AddressFormType {
+    public enum AddressFormType: String, Codable, CaseIterable {
 
         /// Display full address form
-        case full
```

Comment:
Not sure if assigning English strings to cases is a good idea 🤔
It is not expected that merchants will ever use them, however we also not providing localisation to it.
It will become part of public API that we need to maintain..

Maybe we can just keep and capitalize text on DemoApp side?
```swift
ForEach(CardComponent.AddressFormType.allCases, id: \.self) {
        Text($0.rawValuecapitalized)
}
```

---

I appreciate that making enums compatible to `String` - it makes parsing them on React Native easier 😄

Reply:
Maybe instead of using the rawValue we could have a `var displayName: String` that returns the value we want to show in the interface - would make it more readable :)

Reply:
d220da06d51a1b66beebb3f903002cb53f110f60